City-of-Helsinki/drupal-employment-services
# THF-525: Other language articles and events

Test with this UI branch: https://github.com/City-of-Helsinki/employment-services-ui/pull/373

**Install**
- Checkout branch
- Run `drush cim -y;drush cr`
- Delete and reimport events: `drush edel node --bundle=event;drush linkedevents:sync`
- Create some news and newsletter articles on RU, SO and UA languages
- Reindex events and articles: `drush sapi-c;drush sapi-rt;drush sapi-i`

**To test**
- Add the "events list" and "news list" shortlist component to all three landing pages
- Create new landing pages for each language with the full "events list", "news list" (separate ones for news and newsletters) components
- Test that the shortlist and full filterable list components work as intended
  - The shortlist components should be missing the "Show all news" / "Show all events" links because the URLs are set to `null` in common.json for these languages 
- The article and event URL aliases should follow this format: `/[langcode]/frontpage/[title]`
